site/public/js/forum.js

```diff
@@ -0,0 +1,1057 @@
+$(function(){
```

Comment:
We probably should have this within a function (this is for forum categories page correct?). Let's have that page reference this function on load instead of having the load functionality here.

Reply:
implementing this made the categories get checkboxes!